cra/task-com-editor.yaml

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
`the file should be named `task-comm-editor.yaml` to be consistent with the tekton task name